### PR TITLE
discgolf_face initial commit

### PR DESCRIPTION
**Pull request by RighteousLightning**
New watch face, keep score during rounds of discgolf or golf! 

Full description in C file.

### CODE REVIEW

File: movement/watch_faces/complication/discgolf_face.c [hunk 1]
```diff
@@ -0,0 +1,319 @@
+#include <stdlib.h>
+#include <string.h>
+#include "discgolf_face.h"                                      // Remember to change number of courses in this file
+#include "watch.h"
+#include "watch_utility.h"
+
+/*
```
Comment (neutralinsomniac) [inline movement/watch_faces/complication/discgolf_face.c:7]: Thank you for adding documentation! Would you mind moving this comment block into discgolf_face.h as per [template.h](https://github.com/joeycastillo/Sensor-Watch/blob/main/movement/template/template.h)? Thanks!

Comment (RighteousLightning) [reply movement/watch_faces/complication/discgolf_face.c]: No problem! 
Comment block has been moved to the header file and I also added the license block to both files.

**Comment by RighteousLightning** [comment]
Also, if anyone has an idea on how to make the high scores survive a battery change, that would be cool, currently it's initialized with all 99's at boot.

**Comment by ostyn** [comment]
> Also, if anyone has an idea on how to make the high scores survive a battery change, that would be cool, currently it's initialized with all 99's at boot.

For export at least, this would be interesting:
https://github.com/gugray/Sensor-Watch-ChirpyRX

**Pull request merged**